rest_rules.cpp

```diff
@@ -1350,35 +1350,23 @@ bool DeRestPluginPrivate::evaluateRule(Rule &rule, const Event &e, Resource *eRe
         else if (c->op() == RuleCondition::OpIn && c->suffix() == RConfigLocalTime)
         {
             QTime t = now.time();
-            QTime rt;
 
-            if (rule.lastTriggered().isValid())
+            if (rule.lastTriggered().isValid() && rule.lastTriggered().secsTo(now) < 24 * 3600)

```

Comment:
Does this work: `lastTriggered()` is UTC and `now` is local time?

Reply:
Funny enough it did.  Apparently the timezone is only considered when converting to/from a string.

Functionally it doesn't do what I wanted, thought, see below.

Reply:
Interesting, maybe they also use the timezone info from QDateTime class and do some internal conversions.